src/js/visual/configurator-prc.js

```diff
@@ -399,30 +399,35 @@ ripe.ConfiguratorPrc.prototype.resize = async function(size, width, height) {
     }
 
     const area = this.element.querySelector(".area");
-    const frontMask = this.element.querySelector(".front-mask");
     const back = this.element.querySelector(".back");
-    const mask = this.element.querySelector(".mask");
     area.width = width * this.pixelRatio;
     area.height = height * this.pixelRatio;
     area.style.width = width + "px";
     area.style.height = height + "px";
-    frontMask.width = width;
-    frontMask.height = height;
-    frontMask.style.width = width + "px";
-    frontMask.style.height = height + "px";
-    frontMask.style.marginLeft = `-${String(width)}px`;
     back.width = width * this.pixelRatio;
     back.height = height * this.pixelRatio;
     back.style.width = width + "px";
     back.style.height = height + "px";
     back.style.marginLeft = `-${String(width)}px`;
-    mask.width = width;
-    mask.height = height;
-    mask.style.width = width + "px";
-    mask.style.height = height + "px";
     this.currentSize = size;
     this.currentWidth = width;
     this.currentHeight = height;
+
+    if (this.useMasks) {

```

Comment:
I'd say that this should also be passing if `this.useMasks` is null or even undefined

```suggestion
    if (this.useMasks !== false) {

```